test/TestAssets/SimpleTestProject/UnitTest1.cs

```diff
@@ -20,6 +20,7 @@ public class UnitTest1
         [TestMethod]
         public void PassingTest()
         {
+            System.Threading.Thread.Sleep(10000);
```

Comment:
This will unnecessarily increase the time for all the acceptance tests. Should we add another test asset?

Reply:
Done.